dependency-bundles/bom/pom.xml

```
@@ -68,7 +68,7 @@
 		<resilience4j.version>2.2.0</resilience4j.version>
 		<!-- JSON & XML stuff -->
 		<gson.version>2.10.1</gson.version>
-		<jackson.version>2.16.2</jackson.version>
+		<jackson.version>2.17.0</jackson.version>
```

Revert, duplicate #361
```suggestion
		<jackson.version>2.16.2</jackson.version>
```